chromium_src/chrome/browser/ui/toolbar/app_menu_model.cc

```diff
@@ -22,6 +25,7 @@
 #undef BookmarkSubMenuModel
 #undef RecentTabsSubMenuModel
 
+#undef IDS_BRAVE_NEW_PRIVATE_TAB
```

Comment:
I think we don't have to undef this.

Reply:
Yes! But I thought it would be good practice to always `undef` the macro definition in the same file if it's not required beyond that file. If you feel it's odd, I'll remove it.

Reply:
I believe IDS_BRAVE_NEW_PRIVATE_TAB is not the case, as it's defined in `generated resource.h` and should always have the same value across files.

Reply:
Yeah, I understand that. But in general, we never include `.cc` files, right? Hence, it will not affect other files.

That being said -  Since there is a theoretical possibility, I'm a bit convinced by your point, I'll remove it now :+1: